### PR TITLE
Support for mop_14 and bug fix in KpJsonImporter

### PR DESCRIPTION
**Pull request by prachi9**
In KpJsonImporter ReadOne(), we were previously looping over all players for one frame but returning only last player's character pose - this was resulting in sparse rigs. Now, ReadOne() processes a single player from a frame.

Added support for mop_14 which is essentially mop_19 minus last 5 key points.

### CODE REVIEW

File: kp2rig/src/main.cpp [hunk 1]
```diff
@@ -134,7 +134,7 @@ int main( int argc, char *argv[] )
    {
       printf( "capturing from STDIN...\n" );
    }
-
```
Comment (johnhe4) [inline kp2rig/src/main.cpp]: I'd revert (or equivalent) this file

**Review by ghost — DISMISSED** [review]
I've built mop_14 branch and tested with 2 inputs: one type "mop_14" and type "mop_19" both with IPP smoothing enabled and results look ok.

**Review by johnhe4 — COMMENTED** [review]
Recommend removing an unchanged file, otherwise looks good visually

**Review by johnhe4 — DISMISSED** [review]

File: kp2rig/src/main.cpp (hunk 1 above)
Comment (prachi9) [reply kp2rig/src/main.cpp]: Done

**Review by ghost — APPROVED** [review]
looks like the same changes as i tested on first review request before removing one unchanged file.

**Pull request merged**